### DISCUSSION TITLE
lexer for log files

### DISCUSSION THREAD

**Discussion by brynne8** [orbitalquark/scintillua#45] [General]
I created a lexer for log files. It's because my colleagues often copy log files and send back for us to view. We need some highlight to find out errors. [link](https://github.com/AlexanderMisel/ta-tweaks/blob/master/log.lua)

PS:  I added MIT license to my repo so that you can use my code without copyright issues.

**Comment by brynne8** [comment]
The lexer works better when lexer.FUNCTION and lexer.KEYWORD use different colors.

**Comment by orbitalquark** [comment]
Thanks for sharing!
